alea/models/blueice_extended_model.py

```diff
@@ -265,10 +276,10 @@ def __init__(self, parameters: Parameters):
         parameter_list = self.parameters.names
 
         self.constraint_functions = self._get_constraint_functions()
-        super().__init__(
-            func=self.ancillary_likelihood_sum,
-            parameter_list=parameter_list,
-            config=self.parameters.nominal_values)
+        super().__init__(func=self.ancillary_likelihood_sum,
```

Comment:
I somehow particularly prefer the hanging indent, like at the main branch: Hanging indentation is a type-setting style where all the lines in a paragraph are indented except the first line. Because this will reduce the length of lines. 

Reference: https://legacy.python.org/dev/peps/pep-0008/#indentation

Reply:
I don't mind the hanging indent but here it's not needed since the lines are not too long. We can make some pep8 PR at some point but I would like to not go overboard now with style choices in this PR.